cartography/intel/gcp/compute.py

```diff
@@ -200,6 +203,7 @@ def get_https_proxies(compute: Resource, project_id: str, common_job_parameters)
 def transform_https_proxies(proxies: List, project_id: str) -> List[Resource]:
     list_proxies = []
     for proxy in proxies:
+        proxy['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='vpc')
```

Comment:
resource_name seems inaccurate. should this be `compute_instance_vpc_network` or something new?

Reply:
We are redirecting it to the VPC service console not to specific VPC network. As there is no console link for proxy